java-compiler-testing/src/main/java/io/github/ascopes/jct/compilers/SimpleCompiler.java

```diff
@@ -169,6 +170,9 @@ public SimpleCompilation compile() {
     // internally, such as the ECJ global lock that we apply to prevent bugs. Without this, we'd
     // have to mock dozens of additional moving parts. It is difficult to stub super methods
     // if we go down that route.
+	if (compiled)
+		throw new AlreadyUsedCompilerException("There has been a second call to compile() in this Compiler");
```

Comment:
```suggestion
    if (compiled) {
      throw new AlreadyUsedCompilerException("There has been a second call to compile() in this Compiler");
    }
```

Just for consistency (also uses 2-space indents rather than tabs, which is something I think checkstyle will flag).